plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1077,6 +1120,13 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
             } );
 
       my->accepted_block_connection = my->chain->accepted_block.connect( [this]( const block_state_ptr& blk ) {
+         if (auto dm_logger = my->chain->get_deep_mind_logger()) {
+            fc_dlog(*dm_logger, "ACCEPTED_BLOCK ${num} ${blk}",
+               ("num", blk->block_num)
+               ("blk", chain().to_variant_with_abi(blk, abi_serializer::create_yield_function(fc::microseconds(config::dmlog_abi_serializer_max_time_us))))
```

Comment:
ABI serializer is not needed here, the block state does not contain any ABI serializable data, so I will remove that.

Reply:
Done.